templates/question_rate.mustache

```diff
@@ -135,12 +135,18 @@
     {{#qelements.rows}}
     <tr class="raterow">
       {{#cols}}
-      <td {{#colstyle}}style="{{.}}"{{/colstyle}}{{#colclass}} class="{{.}}"{{/colclass}}{{#coltitle}} title="{{.}}"{{/coltitle}} 
+      <td {{#colstyle}}style="{{.}}"{{/colstyle}}{{#colclass}} class="{{.}}"{{/colclass}}{{#coltitle}} title="{{.}}"{{/coltitle}}
 			{{#colinput}} onclick="if('{{id}}'.length>0) document.getElementById('{{id}}').checked = true"{{/colinput}}>
         {{#coltext}}{{{.}}}{{/coltext}}
         {{#colhiddentext}}<span class="accesshide">{{{.}}}</span>{{/colhiddentext}}
-        {{#colinput}}<input type="radio" name="{{name}}" id="{{id}}" value="{{value}}"{{#checked}} checked="checked"{{/checked}}{{#disabled}} disabled="disabled"{{/disabled}}{{#onclick}} onclick="{{.}}"{{/onclick}}>
-        {{#label}}<label for="{{id}}" class="accesshide" aria-label="{{{.}}}">{{{.}}}</label>{{/label}}{{/colinput}}
+          {{#colinput}}
+              {{#label}}
+                  <input type="radio" name="{{name}}" id="{{id}}" value="{{value}}"{{#checked}} aria-label="{{#coltext}}{{{.}}}{{/coltext}}" checked="checked"{{/checked}}{{#disabled}} disabled="disabled"{{/disabled}}{{#onclick}} onclick="{{.}}"{{/onclick}}>
+              {{/label}}
+              {{^label}}
+                  <input type="radio" aria-label="{{#coltext}}{{{.}}}{{/coltext}}" name="{{name}}" id="{{id}}" value="{{value}}"{{#checked}} checked="checked"{{/checked}}{{#disabled}} disabled="disabled"{{/disabled}}{{#onclick}} onclick="{{.}}"{{/onclick}}>
```

Comment:
Is this not a copy of the #label one? Just with the attributes in different positions?

Reply:
so the second input targets the first three radio buttons without labels not the actual "ratings or questions"

Reply:
But the output is essentially the same isn't it? If so, no need for the checks, you can just output the input